### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by codegeek004** [nrnb/GoogleSummerOfCode#244] [Q&A] [answered]
We are a group of 3 people and want to contribute in this project. What do we need to contribute in this project? Can you please assist me with this.

**Comment by khanspers** [comment, marked as answer]
Hi @codegeek004, Feel free to browse our project ideas and complete our [NRNB Academy interest form](http://nrnb.org/academy-admissions.html) if you find something of interest, and we can go from there. We use this project ideas list for our participation in Google Summer of Code, as well as our internal training program, NRNB Academy.

**Reply by codegeek004** [reply]
Sure! We will definitely browse project ideas and complete the interest form. Thank you so much for your response.
